src/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,145 @@
+import { fetchData } from "@/Redux/Features/Auth/ChangePasswordSlice";
+import { LockOpen } from "@mui/icons-material";
+import { LoadingButton } from "@mui/lab";
+import {
+  Button,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  TextField,
+  Typography,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import Modal from "@mui/material/Modal";
+import { useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import { useDispatch, useSelector } from "react-redux";
+
+export default function ChangePassword() {
+  const dispatch = useDispatch();
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const required = "This Field is required";
+
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    getValues,
+    formState: { errors },
+  } = useForm();
+  const onSubmit = (data: { email: string; password: string }) => {
+    dispatch(fetchData(data));
+  };
+  const { isConfirmPassword, loading } = useSelector(
+    (state) => state.changePassword
+  );
+  const [isPasswordChanged, setIsPasswordChanged] = useState(false);
+
+  useEffect(() => {
+    if (isConfirmPassword && !isPasswordChanged) {
+      handleClose();
+      setValue("oldPassword", null);
+      setValue("newPassword", null);
+      setValue("confirmPassword", null);
+    }
+  }, [isConfirmPassword, isPasswordChanged]);
+  return (
+    <>
+      <div>
+        <ListItemButton onClick={handleOpen}>
+          <ListItemIcon>
+            <LockOpen />
+          </ListItemIcon>
+          <ListItemText primary={"ChangePassword"} />
+        </ListItemButton>
+        <Modal
+          open={open}
+          onClose={handleClose}
+          aria-labelledby="modal-modal-title"
```

Comment:
You need to edit the content of the aria labels for better accessibility for people using screen readers.